static/js/courseware_carousel.js

```diff
@@ -27,6 +27,13 @@ $(".course-slider").slick({
         slidesToShow:   1,
         slidesToScroll: 1
       }
+    },
+    {
+      breakpoint: 500,
```

Comment:
The problem appears to be a misconfigured breakpoint. Specifically the `767px` breakpoint. That needs to be fixed because that's the point where the double cards are bunched together in the interface making it look off.

Reply:
@ahmed-arbisoft do we wan to show only one column on `<768px` ?